modules/authwindowslive/www/linkback.php

```diff
@@ -4,30 +4,30 @@
  * Handle linkback() response from Windows Live ID.
  */
 
-if (!array_key_exists('wrap_client_state', $_REQUEST)) {
-	throw new Exception('Lost OAuth-WRAP Client State');
+if (!array_key_exists('state', $_REQUEST)) {
+	throw new Exception('Lost OAuth Client State');
 }
-$state = SimpleSAML_Auth_State::loadState($_REQUEST['wrap_client_state'], sspmod_authwindowslive_Auth_Source_LiveID::STAGE_INIT);
+$state = SimpleSAML_Auth_State::loadState($_REQUEST['state'], sspmod_authwindowslive_Auth_Source_LiveID::STAGE_INIT);
 
 // http://msdn.microsoft.com/en-us/library/ff749771.aspx
-if (array_key_exists('wrap_verification_code', $_REQUEST)) {
+if (array_key_exists('code', $_REQUEST)) {
 
 	// Good
-	$state['authwindowslive:wrap_verification_code'] = $_REQUEST['wrap_verification_code'];
+	$state['authwindowslive:verification_code'] = $_REQUEST['code'];
 
 	if (array_key_exists('exp', $_REQUEST))
-		$state['authwindowslive:wrap_exp'] = $_REQUEST['exp'];
+		$state['authwindowslive:exp'] = $_REQUEST['exp'];
 
 } else {
-	// wrap_error_reason = 'user_denied' means user chose not to login with LiveID
+	// error_reason = 'user_denied' means user chose not to login with LiveID
 	// redirect them to their original page so they can choose another auth mechanism
-	if ($_REQUEST['wrap_error_reason'] === 'user_denied') {
+	if ($_REQUEST['error'] === 'user_denied') {
```

Comment:
It isn't a bug - I didn't update the comment. I've improved the comment now to explain it better.